wdn/templates_4.0/scripts/search.js

```diff
@@ -151,15 +156,49 @@ define(['jquery', 'wdn', 'require', 'modernizr', 'navigation'], function($, WDN,
 					if (postReady && $unlSearch[0].contentWindow.postMessage) {
 						e.preventDefault();
 						$unlSearch[0].contentWindow.postMessage(domQ.val(), searchOrigin);
+						if ('auto' != source) {
```

Comment:
Just FYI, using type-strict comparisons ( `===` , `!==` ) are generally encouraged for performance and to avoid logical accidents.